app/views/sample_types/select.html.erb

```diff
@@ -3,11 +3,22 @@
   projects = Project.all.select{|p| p.sample_types.any? }
   your_projects = projects & current_user.person.projects
   other_projects = projects - your_projects
+	templates_enabled = Seek::Config.sample_type_template_enabled
 %>
-<h1>Select Sample Type</h1>
-<p>
-  Use the filters to find the sample type you are looking for, and then click the New Sample button to create one of this type.
-</p>
+<div class="row contribution-header">
+	<% if templates_enabled %>
+		<div class="col-md-12">
+			<%= link_to "Search by #{t('template').pluralize}", query_form_samples_path, class: "btn btn-default btn-lg" %>
+			<p style="font-size:20px;margin-top:20px">OR</p>
+		</div>
+	<% end %>
+  <div class="col-md-12">
+		<h1>Select <%=t('sample_type') %></h1>
+		<p>
+			Use the filters to find the <%=t('sample_type') %> you are looking for, and then click the New Sample button to create one of this type.
```

Comment:
will this work with `templates_enabled = false` ? Difficult to tell from the diff

Reply:
It won't show the button if `templates_enabled = false`
Redirection from the `sample_qury` page added in case `templates_enabled = false`